package.json

```diff
@@ -77,6 +77,7 @@
     "yarn": "^1.22.19"
   },
   "resolutions": {
+    "**/@hello-pangea/dnd": "16.2.0",
```

Comment:
@tkajtoch @breehall 

I don't love this but I don't see an easy way around this. We were able to update Kibana's base webpack config and Cypress webpack config to use the optional chaining babel plugin on the distributed dnd dependency, but Storybook just won't take my updated rule no matter what I try.

I think it's related to this issue: https://github.com/storybookjs/storybook/issues/14292 and the fact that Kibana is still on Webpack v4. I don't think we can upgrade Kibana past `16.2.0` until it's on Webpack v5.

Reply:
omg https://github.com/storybookjs/storybook/issues/14292#issuecomment-803811848 literally had the fix, I don't know why I didn't try that earlier. Everything fully works (in Storybook, Cypress, etc.) with that totally seemingly unrelated yarn resolution.

#JustJavascriptEcosystemThings